src/Stripe.net/Entities/SourceTransactions/StripeSourceTransactionACHCreditTransfer.cs

```diff
@@ -0,0 +1,16 @@
+using Newtonsoft.Json;
+
+namespace Stripe
+{
+    public class StripeSourceTransactionACHCreditTransfer : StripeEntity
```

Comment:
Same for this property (`StripeSourceTransactionAchCreditTransfer` with lower case "ch").